### DISCUSSION TITLE
[Schedule:File:Shading]How to use this import shading information?

### DISCUSSION THREAD

**Discussion by Magnus1100** [NREL/EnergyPlus#10829] [Q&A]
I'm using [Schedule:File:Shading] part to finish my hourly simulation for dynamic shading, but my import csv file seemingly doesn't work. How do I set up my idf and csv files to make sure they can be read and work correctly.

<img width="461" alt="微信图片_20241120173411" src="https://github.com/user-attachments/assets/5462df4d-ecae-44e1-a94e-21f92fb953ba">
<img width="221" alt="微信图片_20241120173507" src="https://github.com/user-attachments/assets/4235934d-14ac-4692-99f2-cb5702c8449f">

**Comment by rraustad** [comment]
There is an example file SolarShadingTest_ImportedShading.idf and associated shading file SolarShadingTest_Shading_Data.csv which should help set up your file.

    ShadowCalculation,
      Imported,                !- Shading Calculation Method
      Periodic,                !- Shading Calculation Update Frequency Method
      1,                       !- Shading Calculation Update Frequency
      ,                        !- Maximum Figures in Shadow Overlap Calculations
      ,                        !- Polygon Clipping Algorithm
      ,                        !- Pixel Counting Resolution
      ,                        !- Sky Diffuse Modeling Algorithm
      No;                      !- Output External Shading Calculation Results

    Schedule:File:Shading,
      SolarShadingTest_Shading_Data.csv;  !- File Name

**Reply by Magnus1100** [reply]
It worked! Thank you so much for your help!

**Reply by bryonSflowers** [reply]
Hey Magnus. How did you resolve the issue? I am still struggling to get the imported csv. file to be read by EnergyPlus.
